### PR TITLE
More descriptive error when job id is left blank on logs command.

### PR DESCRIPTION
**Pull request by Qard**
This is a fix for the first part of #2. Not sure yet how best to handle http errors for the second part.

### CODE REVIEW

File: logs.go [hunk 1]
```diff
@@ -12,5 +12,9 @@ var cmdLogs = &Command{
 }
 
 func runLogs(cmd *Command, args []string) {
+	if len(args) != 1 {
+		panic("You must supply a job id")
```
Comment (titanous) [inline logs.go:16]: How about:

``` go
fmt.Println("You must supply a job id\n")
cmd.printUsage()
os.Exit(1)
```

Comment (Qard) [inline logs.go:16]: Using panic/recover might be better for getting stack traces while in active development, but I can switch that if you prefer a simpler output. I'd recommend spitting that out on stderr though.

Comment (titanous) [inline logs.go:16]: Okay, we can leave it for now.

**Comment by titanous** [comment]
Thanks!

**Pull request merged**